ppdet/data/source/dataset.py

```diff
@@ -75,6 +75,8 @@ def get_cname2cid(self):
         return self.cname2cid
 
     def get_anno(self):
+        if self.anno_path is None:
+            return
```

Comment:
Why return?

Reply:
In infer.py use 'anno_file = dataset.get_anno()' to get annotation file. In yolov3_mobilenet_v1_voc.yml, anno_path is None and it will raise error in os.path.join(self.dataset_dir, self.anno_path)